app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -46,6 +53,7 @@ function FieldButton({
   callback: () => void;
   disabled?: boolean;
   isLoading: boolean;
+  isSubmit: ButtonTypes;
 }) {
```

Comment:
`isSubmit` sounds like it should be `boolean`, if it can be more than one thing I would call it `buttonType`. However, we only need a regular button and a submit, so I would use the boolean.